script/amazon_polly.py

```diff
@@ -0,0 +1,54 @@
+"""Helper script to update supported languages for Amazone Polly text-to-speech (TTS).
+
+N.B. This script requires AWS credentials.
+"""
+
+from pathlib import Path
+
+import boto3
+from pydantic import BaseModel, Field
```

Comment:
Let's avoid the use of Pydantic here

Reply:
Fixed in 761f2c8db09d4dde5ef9d666a8795ecc4c60a19f

Reply:
Is the rule "no Pydantic in build infrastructure and support scripts" or "no Pyndatic in general"?

Reply:
Code doesn't directly depend on Pydantic. No reason to rely on a dependency if it isn't needed.